playwright.config.ts

```
@@ -1,38 +1,64 @@
       tssc.registry as ImageRegistryType,
       tssc.git as GitType,
       tssc.ci as CIType,
       tssc.tpa || '',
       tssc.acs || ''
-    )
+    ),
   },
 }));
 
+// Create ui projects for UI tests from exported test items
+let uiProjects: any[] = [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)